README.md

```diff
@@ -42,6 +42,10 @@ the following dependencies as well: `graphicsmagick` and `cairo`
     sudo apt-get install graphicsmagick cairo
 
 ### Install `beaker`
+Before installing Beaker, make sure you have a `node_modules` directory in your project folder:
+
+    mkdir node_modules
+
```

Comment:
So when you do an `npm install beaker` in an empty directory, it fails unless `node_modules/` exists already?